test-app/tests/unit/settled-test.js

```diff
@@ -69,20 +69,13 @@ module('settled', function (hooks) {
       return !this.isWaiterPending;
     };
 
-    // In Ember < 2.8 `registerWaiter` expected to be bound to
-    // `Ember.Test` 😭
-    //
-    // Once we have dropped support for < 2.8 we should swap this to
-    // use:
-    //
-    // import { registerWaiter } from '@ember/test';
-    Ember.Test.registerWaiter(this._legacyWaiter);
+    registerWaiter(this._legacyWaiter);
```

Comment:
TIL we support the legacy waiters still

Reply:
yea, and probably will continue to do so until they're deprecated

Reply:
I think the entire legacy test system is still in there...